composer.json

```diff
@@ -27,18 +27,18 @@
     ],
     "require": {
         "php": ">=5.6.4",
-        "illuminate/config": "5.4.*|5.5.*|5.6.*|5.7.*|5.8.*",
-        "illuminate/contracts": "5.4.*|5.5.*|5.6.*|5.7.*|5.8.*",
-        "illuminate/http": "5.4.*|5.5.*|5.6.*|5.7.*|5.8.*",
-        "illuminate/support": "5.4.*|5.5.*|5.6.*|5.7.*|5.8.*",
-        "illuminate/translation": "5.4.*|5.5.*|5.6.*|5.7.*|5.8.*",
-        "illuminate/validation": "5.4.*|5.5.*|5.6.*|5.7.*|5.8.*",
-        "illuminate/view": "5.4.*|5.5.*|5.6.*|5.7.*|5.8.*"
+        "illuminate/config": "~5.4|^6.0",
```

Comment:
Defining version as `~5.4` is not right. It is same as `>=5.4.0 <5.5.0`.
Leave it as it was before or define it as `^5.4` so it will mean `>=5.4.0 <6.0.0`

Reply:
I think it's correct `~5.4` mean minimum version is 5.4 and maximum version is 5.8.
https://getcomposer.org/doc/articles/versions.md#tilde-version-range-

Reply:
https://jubianchi.github.io/semver-check/#/constraint/~5.4